### DISCUSSION TITLE
Week 9 APIs Project Updates

### DISCUSSION THREAD

**Discussion by CJ3315** [USU-ITLS-x265-23-summer/class-repo#175] [General]
**### Project Task List (Completed):**

- I was able to create different sections within my portfolio.
- Icons, fonts, and colors were added. 
- Working buttons for each section. 

**### Next Tasks to be completed:** 
Additions to each section:
- videos 
- Images 
- Text 
Background colors and font headers 
Additional Functionality: 
-Submit contact info 
-Request additional info  
![image](https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/133183018/54afd5aa-f343-46a0-a75f-adab05494001)

**Comment by kassyoveson** [comment]
Great job CJ. Keep  going!

**Comment by coopsand** [comment]
I like your UI for your contact form. Very sleek and functional. Can't wait to see what you accomplish next.

**Comment by derekscholes** [comment]
Do you by chance have a link to your GitHub page or a published site? I would love to take a look at your html and css to help me update my own form since mine is kind of hideous.

**Reply by CJ3315** [reply]
https://github.com/CJ3315/Portofolio-Project 
So far I just have different sections. A great resource that I used is font awesome for good-looking icons: https://fontawesome.com/search?q=photo&o=r
Pro tip to create an account!
